### DISCUSSION TITLE
Button to go to a specific expression number

### DISCUSSION THREAD

**Discussion by jared-hughes** [DesModder/DesModder#842] [Ideas]
Helpful for MASSIVE graphs when scrolling just doesn't cut it.

User story: ctrl+F for the expression, but realize that the one after is desired. Press ctrl+G and enter that expression number to scroll there real quick.

Plugin should also:
- automatically open containing folder, if any

Ctrl+G is what Atom and VS Code use, but some people use ctrl+G to search. Is there a better shortcut idea?

**Comment by jared-hughes** [comment]
Perhaps a better idea: Inside the search results, add a button to scroll to expression. Greatly simplifies the whole process.

**Comment by spencerpogo** [comment]
It would be cool to have something like VS Code's ctrl+g popup for desmos. In VS Code, a box pops up in the top center and as you type it scroll to whatever line number you are typing even before you submit (e.g. 363 scrolls to 3, then 36, then 363) and then enter closes the box. 

Technically, this is feasible: 
```js
const n = getUserInput();
const id = Calc.controller.getItemModelByIndex(n-1).id;
Calc.controller.dispatch({ type: "move-focus-to-item", id, where:"end" });
```
